app/models/transaction.rb

```diff
@@ -111,8 +122,16 @@ def self.amount_eq(amount)
     where('abs(amount_cents) = ?', Money.new(amount.to_d.abs * 100).cents)
   end
 
+  def self.category_id_eq(id)
```

Comment:
@gataniel not sure why do you need this. this filter should be handled by ransack